### PR TITLE
implemented the marvin 'o' command

### PR DESCRIPTION
**Pull request by fhanrath**
Integrated the comments

### CODE REVIEW

File: game/world/world.cpp [hunk 1]
```diff
@@ -117,6 +117,22 @@ void World::insertPlayer(std::unique_ptr<Npc> &&npc, std::string_view waypoint)
     game.script()->setInstanceNPC("HERO",*npcPlayer);
   }
 
+void World::setPlayer(Npc* npc) {
+  if(npc==nullptr)
+    return;
+  npcPlayer->setProcessPolicy(Npc::ProcessPolicy::AiNormal);
+  npcPlayer->resumeAiRoutine();
```
Comment (Try) [inline game/world/world.cpp:124]: That is still deferred, now - just wrapped :)
You can have design, like:
```c++
void Npc::resumeAiRoutine(){ /* actual work */ }
```
and
```c++
case AI_ContinueRoutine:
  resumeAiRoutine();
  break;
```

Comment (fhanrath) [reply game/world/world.cpp:124]: Thanks for your patience and help :)

**Comment by Try** [comment]
Find a bug, while testing the last revision: if your character, `npc1` to name it, is dead, and you take control of npc2, then `resumeAiRoutine` will resurrect `npc`.
Other thing seems to be fine.

**Comment by Try** [comment]
Merged, thanks!

**Pull request merged**